client/src/component/admin/__test__/MenuForm.test.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+
+import userEvent from "@testing-library/user-event";
+
+import MenuForm from "../MenuForm";
+
+describe("should render form and form element", () => {
+  it("should render item name input", () => {
+    render(<MenuForm values={{ itemname: "", price: "", categoryId: "" }} />);
+
+    const inputNode = screen.getByTestId("item-name");
```

Comment:
I had previously tried with `getByLabelText('Item name')` got the error. So I switched to `getByTestId`.  I assume,  may be because of using chakra UI component.
![Screenshot 2022-11-17 at 6 18 09 PM](https://user-images.githubusercontent.com/56571107/202601583-76282d07-e4d6-4dd0-8a30-362c10292570.png)
![Screenshot 2022-11-17 at 6 19 03 PM](https://user-images.githubusercontent.com/56571107/202601688-58de065d-05a9-4a77-af7c-f4af77d90f6c.png)

Reply:
Ah yeah looks like since it's required, there are extra elements in the label. You can use a regex instead:

```js
getByLabelText(/Item name/)
```

Reply:
it worked. Thank you